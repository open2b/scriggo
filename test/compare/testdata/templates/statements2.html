{# render #}

{%% var a = 5 %%}
a = {{ a }}
{%%
    var b = a + 1 %%}
b = {{ b }}
{%% var c = b + 1
%%}
c = {{ c }}
{%%var d = c * 2 %%}
d = {{ d }}
{%%
    var e int
    if a == 5 {
        e = 3
    } else {
        e = 2
    }
    e *= 2
%%}
e = {{ e }}

{% macro A %}A{% end %}
{%% show A() %%}

{%%
    if e == 6 {
        show e+4
    }
%%}


