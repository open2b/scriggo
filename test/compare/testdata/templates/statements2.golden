a = 5

b = 6

c = 7

d = 14
e = 6


A
